people.md

```diff
@@ -5,9 +5,34 @@ permalink: /people/
 ---
 {%- for i in site.data.people -%}
 {%- if i.members.size > 0 -%}
-# {{ i.title }}
-{%- for j in i.members %}
-[{{ j.name }}]({{ j.link }}), {{ j.affiliation }}
-{% endfor -%}
+## {{ i.title }}
+<table style="table-layout:fixed; width:100%; border: none;">
+<tbody style="background: none;">
+<tr style="background: none;">
+{% assign k = 0 -%}
+{%- for j in i.members -%}
+{%- if j.img -%}
+{%- capture img -%}{{ j.img }}{%- endcapture -%}
+{%- else -%}
+{%- capture img -%}/assets/images/people/null.jpg{%- endcapture -%}
+{%- endif -%}
+{%- capture name -%}{{ j.name }}{%- endcapture -%}
+{%- capture link -%}{{ j.link }}{%- endcapture -%}
+{%- capture affiliation -%}{{ j.affiliation }}{%- endcapture -%}
+{%- include person.html img=img name=name link=link affiliation=affiliation -%}
```

Comment:
I think these `capture`s could be `assigns`. Is this needed to pass variables to `include`? E.g., does
```
{%- include person.html img=img name=j.name link=j.link affiliation=j.affiliation -%}
```
not work?